site/src/components/layout.md

```diff
@@ -0,0 +1,148 @@
+
+How do I apply it? `layout-holy-grail` to the `layout` element.
+
+Required elements: all, except `layout-body-hero`.
+
+Allowed elements: all.
+
+```Text
+  ┌────────────┐
+  │ Header     │
+  ├────────────┤
+  │ Hero       │
+  ├────────────┤
+  │ Menu       │
+  ├────────────┤
+  │ Main       │
+  ├────────────┤
+  │ Aside      │
+  ├────────────┤
+  │ Footer     │
+  └────────────┘
+```
+
+The specification for this layout is as follows.
+
+| Screensize          | Behavior                  |
+| ------------------- | ------------------------- |
+| Narrow - Widescreen | All elements are stacked. |
+
+### Holy Grail layout
+
+Named after a layout that [historically was difficult to implement](<https://en.wikipedia.org/wiki/Holy_grail_(web_design)>) the holy grail is stacked on narrow widths and progressively shows up to three columns as the screen widens.
+
+How do I apply it? `layout-holy-grail` to the `layout` element.
+Required elements: all, except `layout-body-hero`.
+Allowed elements: all.
+
+The following block is arranged from narrow widths on the left to wider widths on the right.
+
+```Text
+  ┌────────────┐ ┌──────────────┐ ┌──────────────────────┐
+  │ Header     │ │Header        │ │ Header               │
+  ├────────────┤ ├──────────────┤ ├──────────────────────┤
+  │ Hero       │ │Hero          │ │ Hero                 │
+  ├────────────┤ ├─────┬────────┤ ├─────┬────────┬───────┤
+  │ Menu       │ │Menu │ Main   │ │Menu │ Main   │ Aside │
+  ├────────────┤ │     │        │ │     │        │       │
+  │ Main       │ │     │        │ │     │        │       │
+  ├────────────┤ │     ├────────┤ │     │        │       │
+  │ Aside      │ │     │ Aside  │ │     │        │       │
+  ├────────────┤ ├─────┴────────┤ ├─────┴────────┴───────┤
+  │ Footer     │ │Footer        │ │ Footer               │
+  └────────────┘ └──────────────┘ └──────────────────────┘
+```
+
+The specification for this layout is as follows.
+
+| Screensize | Behavior                                                                                                                      |
+| ---------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| Narrow     | All elements are stacked.                                                                                                     |
+| Tablet     | Menu and main are side by side. Main is wider than menu. Aside has a collapsed height and is tucked under main.               |
+| Desktop    | Menu, main, aside are side by side. Main is widest than menu.                                                                 |
```

Comment:
```suggestion
enu, main, aside are side by side. Main is wider than menu.    
```

Reply:
Diff with suggestion looked a bit funny so I committed this change directly.